src/palette/view/widgets/timedialog.cpp

```diff
@@ -148,10 +148,14 @@ void TimeDialog::save()
 //   zChanged
 //---------------------------------------------------------
 
-void TimeDialog::zChanged(int val)
+void TimeDialog::zChanged()
 {
-    Q_UNUSED(val);
-    Fraction sig(zNominal->value(), denominator());
+    int numerator = zNominal->value();  
+    int denominator = this->denominator();  
```

Comment:
Still a space to much after these 2 lines

Reply:
> Still a or 2 trailing space to much after these 2 lines

I did the required changes but it doesn't seem to resolve the issue

Reply:
No, you didn't, you (needlessly) removed some empty lines, but not the trailing spaces in the above 2 lines

Reply:
Ohh i am sorry completey misunderstood it and as i am a  beginner with open source faced some issues with the squash commits part